cmd/dbaas.go

```diff
@@ -153,3 +154,18 @@ func dbaasShowSettings(settings map[string]interface{}) {
 		t.Append(row)
 	}
 }
+
+func dbaasGetType(ctx context.Context, name, zone string) (string, error) {
+	dbs, err := cs.ListDatabaseServices(ctx, zone)
+	if err != nil {
+		return "", fmt.Errorf("failed to retrieve database type: %s", err)
```

Comment:
It doesn't cost us anything to wrap errors (`%w`). Might come in handy in the future.